### PR TITLE
Response group

### PR DESCRIPTION
**Pull request by ghost**
@project-signal @ricardo-quinones basic models added, will work on it once i get a better feel for the app.

### CODE REVIEW

**Comment by signalplus-dev** [comment]
Make sure to grab the latest master branch and rebase your branch off of it.

File: app/models/listen_signal.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# == Schema Information
```
Comment (signalplus-dev) [inline app/models/listen_signal.rb:1]: Any reason why we don't call this `Signal`?

File: db/migrate/20160417200721_create_listen_signals.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class CreateListenSignals < ActiveRecord::Migration
+  def change
+    create_table :listen_signals do |t|
+      t.references :brand, index: true, foreign_key: true
+      t.string :provider
```
Comment (signalplus-dev) [inline db/migrate/20160417200721_create_listen_signals.rb]: Instead we can do a reference to the identity model.

File: db/migrate/20160417201140_create_response_groups.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreateResponseGroups < ActiveRecord::Migration
+  def change
+    create_table :response_groups do |t|
+      t.references :brand, index: true, foreign_key: true
```
Comment (signalplus-dev) [inline db/migrate/20160417201140_create_response_groups.rb]: Any reason we're tying this to brand? We can get that through the signal if needed.

**Comment by ghost** [comment]
@ricardo-quinones  going ahead n merging! fixed all the changes

**Pull request merged**